### ISSUE TITLE
"Open with" from file system freaks out all the menus

### ISSUE THREAD

**Issue by MaanooAk** [notepadqq/notepadqq#409]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/notepadqq/notepadqq/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

![screenshot from 2017-03-25 11-45-35](https://cloud.githubusercontent.com/assets/6997990/24321276/fabac21a-1150-11e7-95dd-be2c4b4d54ab.png)

(All the menus are missing)

**Expected behavior**

![screenshot from 2017-03-25 11-45-55](https://cloud.githubusercontent.com/assets/6997990/24321277/fcd980cc-1150-11e7-8482-f4926151ae76.png)

**Minimal reproduction of the problem with instructions**

- Open notepadqq.
- R-click on file (nautilus) -> open with notepadqq
- All the menus are missing

**Please tell us about your environment:**

Ubuntu 16.04, Notepadqq v1.0.1

**Comment by danieleds** [comment]
I suspect this issue is something which has to do with Ubuntu and Qt, not with Notepadqq itself.

Also, I'm not able to reproduce it on Ubuntu 16.10, Qt 5.6.1

**Comment by MaanooAk** [comment]
The thing is, this start happening after the last Notepadqq update not form the beginning, and my system hasn't change.
Keep this open in case someone else can reproduce it and then we'll now for sure...

**Comment by Teklad** [comment]
Try rolling back just notepadqq to the previous version.  If that doesn't work one of the other updates that slipped in with Nqq is likely a culprit.

**Comment by danieleds** [comment]
@MaanooAk Are you using a pre-release version (from git) or a prepackaged one?

This issue is not completely new to me, but I can't remember when or where I had to deal with it.

Some possibly related refs:
https://bugs.launchpad.net/ubuntu/+source/appmenu-qt5/+bug/1307619
https://forum.qt.io/topic/7276/menu-not-showing-up-in-menubar/11
https://github.com/robotology/yarp/issues/521

**Comment by fxarte** [comment]
First thanks for the app!

I'm seeing the same behavior (Also Ubuntu 16.04, Notepadqq v1.0.1).
To temporarily fix it, I close and reopen Notepadqq. 

I noticed that the windows title had the file name of the current tab, and it was all across the menu bar, apparently being on top the menu, and effectively hiding it.

Here are screenshots with and without the menu:
![image](https://cloud.githubusercontent.com/assets/841166/25189647/c7c559e6-24f7-11e7-93b8-b5d5b148c1f8.png)
reopened:
![image](https://cloud.githubusercontent.com/assets/841166/25189785/3551da7a-24f8-11e7-9085-4372136a56cd.png)

**Comment by fxarte** [comment]
The solution from https://github.com/robotology/yarp/issues/521 fixed it for me:
`sudo apt-get remove --purge appmenu-qt5`

results:
![image](https://cloud.githubusercontent.com/assets/841166/25190588/b13b453e-24fa-11e7-937b-7691e20efa0e.png)

**Comment by JuBan1** [comment]
Nice detective work! I've suspected something outside Nqq is going on but googling didn't net me any useful results and Daniel's Ubuntu packages looked fine.

**Comment by danieleds** [comment]
Yes, this looks like an issue with Qt integration in Ubuntu, as it happens with other applications too.

What makes me think is why it only happens when opening files from Nautilus. This *could* indicate that we may be able to write a workaround.

**Comment by fxarte** [comment]
Hopes this adds extra information, running:
`$> notepadqq Desktop/1.txt` > Start and open 1.txt, menu shows fine, then
`$> notepadqq Desktop/2.txt` > to add 2.txt to running instance, menus are broken

Both from Terminal and XTerm

**Comment by NeoBeum** [comment]
> @JuBan1 [https://github.com/notepadqq/notepadqq/issues/409#ref-issue-228288973](url)
> The solution from robotology/yarp#521 fixed it for me:
sudo apt-get remove --purge appmenu-qt5

Similar yes, but I never had appmenu-qt5 installed.
